src/components/graph/graph.config.js

```diff
@@ -229,6 +230,7 @@ export default {
     linkHighlightBehavior: false,
     maxZoom: 8,
     minZoom: 0.1,
+    initialZoom: 1,
```

Comment:
Just curious here, why should the default value be `1` instead of `0`? I think this value should be `null` by default.